src/site/content/en/blog/image-policies/index.md

```diff
@@ -0,0 +1,273 @@
+whose intrinsic resolution is more than X times larger than the container size
+in either dimension will be replaced with a placeholder image.
+
+
+#### Why?
+
+Serving images larger than what the viewing device can render&mdash;for example,
+serving desktop images to mobile contexts, or serving high-pixel-density images
+to a low-pixel-density device&mdash;is wasting network traffic and device
+memory. Read [Serve images with correct
+dimensions](https://web.dev/serve-images-with-correct-dimensions/) and [Serve
+responsive images](https://web.dev/serve-responsive-images/) for information on
+optimizing your images.
+
+#### Examples
+
+A few examples illustrate this. The following shows the default behavior when cutting an image's display size in half.
+
+![The default resizing behavior](default-100x100.png)
+
+If I apply the following feature policy, I get a placeholder image instead.
+
+`Feature-Policy: oversized-images *(2);`
+
+![When the image is too large for the container](resize-both-dimensions.png)
+
+I get similar results if I lower only the width or the height.
+
+![Resized width](resize-width.png) ![Resized height](resized-height.png)
+
+#### How to use
+
+If you are new to feature policy, please check out [Introduction to Feature
+Policy](https://developers.google.com/web/updates/2018/06/feature-policy) for
+more details.
+
+To summarize, `oversized-images` policy can be specified through either:
+
+* `Feature-Policy` HTTP header
+* `<iframe>` `allow` attribute
+
+To declare the `oversized-images` policy, you need to provide:
+
+* The feature name, `oversized-images` (Required)
+* A list of origins (Optional)
+* The threshold values (i.e., the downscaling ratio X) for the origins, specified in
+  parenthesis (Optional)
+
+We recommend a downscaling ratio of 2.0 or lower. Consider using [responsive
+images](https://web.dev/serve-responsive-images/) with different resolutions to
+best serve images on various screen sizes, resolutions, etc..
+
+#### More examples
+
+`Feature-Policy: oversized-images *(2.0)`
+
+The policy is enforced on all origins with a threshold value of 2.0. Any
+`<img>` element with an image whose downscaling ratio that is greater than
+2.0 is disallowed and will be replaced with a placeholder image.
+
+`Feature-Policy: oversized-images *(inf) 'self'(1.5)`
+
+The policy is enforced on the site's origin with a threshold value of 1.5.
+`<img>` elements in top-level browsing contexts and same origin nested
+browsing contexts will only render normally if the downscaling ratio is less
+than or equal to 1.5. `<img>` elements everywhere else will render normally.
+
+### unoptimized-{lossy,lossless}-images
+The `unoptimized-lossy-images`, `unoptimized-lossless-images`,
+`unoptimized-lossless-images-strict` feature policies restrict the file
+size of an image in relation to its intrinsic resolution:
+
+<dl>
+  <dt><code>unoptimized-lossy-images</code></dt>
```

Comment:
I was wondering if these might look better as bolded text instead of using the code block here.